kinetic/distribution.yaml

```diff
@@ -2328,6 +2328,16 @@ repositories:
       url: https://github.com/ct2034/dockeROS.git
       version: master
     status: developed
+  doosan-robot:
```

Comment:
@tfoote are repo names allowed to have `-`? I only see repos with underscores `_`, but it doesn't appear to be forbidden by by http://www.ros.org/reps/rep-0141.html

Reply:
Do I need more work to be added to the index of 'rosdistro'?

Reply:
@sloretz It's not technically required, often it's a repo that has a single packages and reflects that name and we use underscores in all our naming so that's the default pattern for people.

Reply:
> Before this change can be accepted the permissions changes to all other files need to be removed from the branch.

What additional work should I do about permission?
Can I leave only the modified distribution.yaml file?

The information I commit is: 
`upstream` repository: https://github.com/doosan-robotics/doosan-robot.git
`release` repository: https://github.com/doosan-robotics/doosan-robot-release.git
`distro` file: kinetic/distribution.yaml

Thank you for comment.